doc/rst/language/spec/task-parallelism-and-synchronization.rst

```diff
@@ -471,13 +471,20 @@ it is passed as an actual argument to the task function and all
 references to the field within the task function implicitly refer to the
 corresponding shadow variable.
 
-Each formal argument of a task function has the default argument intent
-by default. See also :ref:`The_Default_Intent`. Note that the default
-intent usually allows the compiler to assume that the value will not be
-concurrently modified. For variables of primitive and class types, this
-has the effect of capturing the value of the variable at task creation
-time and referencing that value instead of the original variable within
-the lexical scope of the task construct.
+The implicit formals of task functions generally have
+:ref:`the default argument intent <The_Default_Intent>` by default. Note that
+the default intent usually allows the compiler to assume that the value will
+not be concurrently modified. For variables of primitive and class types, this
+has the effect of capturing the value of the variable at task creation time and
+referencing that value instead of the original variable within the lexical
+scope of the task construct.
```

Comment:
This is language that Michael added for the cvref change, I don't think that should be changed in this PR.

Reply:
I suspect we have had this language since the introduction of the original default task intent.

@mppf what do you think about removing "this has the effect of capturing the value of the variable at task creation time," now that `const` requires the outer variable to remain unchanged?

Reply:
How about we change it from this

> For variables of primitive and class types, this
has the effect of capturing the value of the variable at task creation time and
referencing that value instead of the original variable within the lexical
scope of the task construct.

to this:

> That assumption is useful for the compiler to, for example, make a per-task copy of a captured variable of `int` type.

Reply:
Works for me, except I prefer "captured" --> "outer" or similar.

Reply:
I made this change in https://github.com/chapel-lang/chapel/pull/24135/commits/f811e52f312dbd151cd5b3bdd1950ad6266e719a